### ISSUE TITLE
BDDfy.html does not write to bin folder 

### ISSUE THREAD

**Issue by MickeyFinn** [TestStack/TestStack.BDDfy#237]
Developing on Windows 7 machine, using Visual Studio 2015, .NET Core 
The BDDfy.html report doesn't output to test project bin folder as expected but instead to C:/Users/{user}/.nuget/packages/TestStack.BDDfy/4.3.1/lib/netstandard1.5/BDDfy.html
interestingly enough the BDDfy.md & Diagnostics.json files actually do write to the bin/debug/netcoreapp1.0/ folder as expected

**Comment by mwhelan** [comment]
Hi, @MickeyFinn . Yes, that was an unfortunate side effect from adding support for .Net Core. It seemed to be a "feature" of the System.AppContext library that we took a dependency on but on closer inspection as to why it only happens to the HTML report, as you have pointed out, I think I have found a coding error on my part. Some copy and paste code in two places where only one was updated. (Hanging my head in shame). :-) I've submitted a fix. Hopefully we can release it soon.
